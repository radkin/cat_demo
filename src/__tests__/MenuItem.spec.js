import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import MenuItem, { Counter, dataReducer } from '../Hamburger/MenuItem';

describe('MenuItem', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<MenuItem />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
