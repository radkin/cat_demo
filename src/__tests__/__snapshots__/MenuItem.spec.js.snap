// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem snapshot renders 1`] = `
<div
  style={
    Object {
      "animation": "1s appear forwards",
      "animationDelay": undefined,
      "opacity": 0,
    }
  }
>
  <div
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "animation": "0.5s slideIn forwards",
        "animationDelay": undefined,
        "color": "#fafafa",
        "cursor": "pointer",
        "fontFamily": "'Open Sans', sans-serif",
        "fontSize": "1.2rem",
        "margin": "0 5%",
        "padding": "1rem 0",
        "transition": "color 0.2s ease-in-out",
      }
    }
  />
  <div
    style={
      Object {
        "animation": "0.5s shrink forwards",
        "animationDelay": undefined,
        "background": "gray",
        "height": "1px",
        "margin": "0 auto",
        "width": "90%",
      }
    }
  />
</div>
`;
